docs/_includes/layouts/partials/side-navigation.njk

```diff
@@ -0,0 +1,30 @@
+{% macro appSideNavigation(params) %}
```

Comment:
Should this file live in `_includes/partials` instead of `_includes/layouts/partials`?
It's not a layout

Reply:
keeping it in `_includes/layouts/partials` as the side nav is used on layouts